webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+
+const config = {
+  entry: path.join(__dirname, '/client/components/index.ts'),
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        loader: 'babel-loader',
+        exclude: /node_modules/,
+      },
+      {
+        test: /\.css$/i,
+        use: ['css-loader'],
+      },
+      {
+        test: /\.s[ac]ss$/i,
+        use: ['css-loader', 'sass-loader'],
+      },
+    ],
+  },
+  resolve: {
+    extensions: ['.tsx', '.ts', '.js'],
```

Comment:
does `.scss` and `.css` need to be added here? Or, are they not entrypoints so the `module.rules` takes care of it?

Reply:
I don't think they need to be added because when styles are imported the extension is used.